### PR TITLE
finished function to load userData to profile page

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: src/components/profile/UserInfo.js [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+  const [userData, setUserData] = useState(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      if (!token) {
+        console.error("No token found. Please log in.");
+        return;
+      }
+      try {
+        const response = await axios.get("http://localhost:5005/auth/user", {
```
Comment (travistrle) [inline src/components/profile/UserInfo.js]: Why do you hard code this url?

File: src/components/profile/UserInfo.js [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
 
 export default function UserInfo() {
-  return <SelectedListItem list={user} />;
+  const token = localStorage.getItem("token");
+  const [userData, setUserData] = useState(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      if (!token) {
+        console.error("No token found. Please log in.");
```
Comment (travistrle) [inline src/components/profile/UserInfo.js]: You shouldn't use console, instead using toast.

File: src/components/profile/UserInfo.js [hunk 3]
```diff
@@ -1,18 +1,51 @@
...
+          email: response.data.email,
+          city: response.data.city,
+          country: response.data.country,
+        };
+
+        setUserData(userInfor);
+      } catch (error) {
+        console.error(
+          "Error fetching user data:",
+          error.response || error.message
```
Comment (travistrle) [inline src/components/profile/UserInfo.js:42]: same right here. And avoid show plain error message to user

File: src/components/profile/UserInfo.js (hunk 1 above)
Comment (tracytrle) [reply src/components/profile/UserInfo.js]: fixed

File: src/components/profile/UserInfo.js (hunk 2 above)
Comment (tracytrle) [reply src/components/profile/UserInfo.js]: fixed

**Review by travistrle — APPROVED** [review]

File: src/components/profile/UserInfo.js (hunk 3 above)
Comment (tracytrle) [reply src/components/profile/UserInfo.js:42]: fixed

**Pull request merged**